src/main/java/uk/gov/hmcts/reform/waworkflowapi/models/Task.java

```diff
@@ -0,0 +1,114 @@
+package uk.gov.hmcts.reform.waworkflowapi.models;
+
+@SuppressWarnings({"PMD.TooManyFields", "PMD.ShortClassName", "PMD.UnnecessaryConstructor", "PMD.UncommentedEmptyConstructor"})
+public class Task {
+
+    private String id;
+    private String name;
+    private String assignee;
+    private String created;
+    private String due;
+    private String followUp;
+    private String delegationState;
+    private String description;
+    private String executionId;
+    private String owner;
+    private String parentTaskId;
+    private int priority;
+    private String processDefinitionId;
+    private String processInstanceId;
+    private String taskDefinitionKey;
+    private String caseExecutionId;
+    private String caseInstanceId;
+    private String caseDefinitionId;
+    private boolean suspended;
+    private String formKey;
+    private String tenantId;
+
+    public Task() {
+    }
+
+    public String getId() {
```

Comment:
you can get rid of this boilter plate code by just adding the lombok annotation @Getter see here https://www.baeldung.com/intro-to-project-lombok